ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_common.py

```diff
@@ -186,3 +186,59 @@ def test_paddle_zeropad2d(
         padding=padding,
         data_format=dataformat,
     )
+
+
+# linear
+@st.composite
+def x_and_linear(draw, dtypes):
+    dtype = draw(dtypes)
+    in_features = draw(helpers.ints(min_value=1, max_value=1))
+    out_features = draw(helpers.ints(min_value=1, max_value=1))
```

Comment:
We should be testing for more than one features. I think you should just import the `x_and_linear` strategy that is used in the torch frontend's test_linear_functions.